README.md

```diff
@@ -40,7 +40,11 @@ See examples below for usage.
 
 ### Alpine Repository (recommended)
 
-Configure the alpine repository
+[Cloud Posse](https://cloudposse.com) provides a public Alpine repository that is hosted on Amazon S3 and fronted by [CloudFlare's CDN](http://cloudflare.com) with end-to-end TLS. This ensures insane availability with DDoS mitigation and low-cost hosting. This is ultimately more reliable than depending on [GitHub for availability](https://twitter.com/githubstatus) and provides an easier way to manage dependencies pinned at multiple versions. 
+
+The repository itself is managed using [`alpinist`](https://github.com/cloudposse/alpinist), which takes care of the heavy lifting of building repository indexes. You can self-host your own Alpine repository using this strategy.
+
+Configure the alpine repository:
```

Comment:
I'm not sure if this is relevant to the current PR, but I believe it would be great to add an example of using apk setup/package install in Dockerfile. The following code snippet is not working on "vanilla" alpine-based Dockerfile (as "curl", " bash" should be installed first, and sudo is not needed, etc).